src/topp/OpenSwathChromatogramExtractor.cpp

```diff
@@ -52,11 +52,11 @@ using namespace std;
 #include <omp.h>
 #endif
 
-#ifdef _OPENMP
-  #define IF_MASTERTHREAD if (omp_get_thread_num() ==0)  
-#else
-  #define IF_MASTERTHREAD 
-#endif    
+// #ifdef _OPENMP
```

Comment:
remove?!
The code using it is commented out anyways (and seems wrong anyways -- so delete as well?)